db/migrate/20210309025913_add_title_to_students.rb

```diff
@@ -0,0 +1,5 @@
+class AddTitleToStudents < ActiveRecord::Migration
+  def change
+    add_column :students, :title, :string
```

Comment:
you should into `rake db:rollback`? this will allow you to undo migrations and allow you to add this into the original create_table and remove the need for this separate migration.
It's okay to rollback migrations as long as it's the same PR.